demos/handwritten_text_recognition_demo/python/handwritten_text_recognition_demo.py

```diff
@@ -77,16 +78,16 @@ def main():
     log.info('OpenVINO Inference Engine')
     log.info('\tbuild: {}'.format(get_version()))
     ie = IECore()
+    ie.set_config(config={"GPU_ENABLE_LOOP_UNROLLING": "NO", "CACHE_DIR": "./"}, device_name="GPU")
```

Comment:
why this config become required for running demo? Why you use current directory as cache?

Reply:
Hi, this config is to speed up loading the network on GPUs, without the config, loading the model is very time consuming.

Reply:
but, what if user does not have intel gpu on running machine? I think it will be better to enable it only if GPU selected as running device

Reply:
That would be hard to handle all cases when gpu may be used with this auto plugin and stuff. `device_name` arg already sets it only for gpu. I would leave it as it is for now.